src/mappings/oba-efo-mapping-exclusions.sssom.tsv

```diff
@@ -1,2 +1,6 @@
 subject_id	subject_label	predicate_id	object_id	match_type	comment
-EFO:0005514	HER2 status	skos:exactMatch	sssom:NoMapping	HumanCurated	tumor level measurements out of scope for OBA
\ No newline at end of file
+EFO:0005514	HER2 status	skos:exactMatch	sssom:NoMapping	HumanCurated	tumor level measurements out of scope for OBA

```

Comment:
It is important that this is excellent suggestion is communicated in your meetings with GWAS.